terminus-ui/tsconfig.spec.json

```diff
@@ -7,6 +7,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "types": [
+      "jsdom",
```

Comment:
w/o specifying this, what do we lose?

Reply:
I'm honestly not 100% sure. I'd assume at the very least we should get better typings when dealing with DOM in tests.

There was nothing explicit it was added for, but as I was looking through various github repos and tutorials trying to track down a fix for that odd DOM error I was fighting I saw this and figured it should be a good addition since we're definitely using JSDom and this `tsconfig` is only used for tests.

It didn't solve my issue, but I left it as it probably is correct to include since we are using JSDom.